features/cells/src/main/java/com/wire/android/feature/cells/ui/LoadingScreen.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Wire
+ * Copyright (C) 2025 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.android.feature.cells.ui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.semantics.invisibleToUser
+import androidx.compose.ui.semantics.semantics
+import com.wire.android.ui.common.colorsScheme
+import com.wire.android.ui.common.dimensions
+import com.wire.android.ui.common.progress.WireCircularProgressIndicator
+
+@Composable
+fun LoadingScreen(
+    modifier: Modifier = Modifier,
+) {
+    Box(
+        modifier = modifier.fillMaxSize(),
+        contentAlignment = Alignment.Center
+    ) {
+        WireCircularProgressIndicator(
+            modifier = Modifier.size(dimensions().spacing32x),
+            progressColor = colorsScheme().primary
+        )
+    }
+}
+
+@Composable
+fun FullScreenLoading(
+    modifier: Modifier = Modifier,
+) {
+    Box(
+        modifier = modifier
+            .fillMaxSize()
+            .pointerInput(Unit) {}
+            .semantics { invisibleToUser() }
+            .background(Color.Gray.copy(alpha = 0.6f)),
```

Comment:
```suggestion
            .background(colorsScheme().scrim),
```
please use `colorsScheme().scrim`

Reply:
this scrim value has 0.55 as aplha value

Reply:
@ohassine can you show me the design for it with alpha 0.6?
I'm pretty sure if there is one then it's wrong because we are supposed to use only colors from design documentation and our color scheme, so if there was need for some grey color with alpha 0.6 then it should be first added to the documentation and our color scheme and it shouldn't be hardcoded `Color.Gray`